### PR TITLE
Fix issue with returning incorrect value for string inputs

### LINKED ISSUE #192: Fibonacci solution bug
Current fibonacci solution returns `1` instead of `0` when passed `"0"`, because it uses strict equality.

https://github.com/TheOdinProject/javascript-exercises/blob/8150be944bc01cb3bea5b50a760ae88e405693c1/fibonacci/fibonacci.js#L3

I'm opening this issue because tests state that the function should accept a string representation of a number.

### PR DESCRIPTION
**Pull request by Nitinbhojsingh**
Issue Fix #192 

I fixed the issue in the fibonacci function where it was returning 1 instead of 0 for string inputs that represent the number 0. I did this by adding a line of code that converts the input to a number and then checking if it's NaN or less than 0. This change should make the function work correctly for both number inputs and string inputs that represent valid numbers.

### CODE REVIEW

**Review by Nitinbhojsingh — COMMENTED** [review]
I have recently made some changes to the fibonacci function in my forked repository, and I would like to request a review from you.

File: 10_fibonacci/solution/fibonacci-solution.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 const fibonacci = function (count) {
-  if (count < 0) return "OOPS";
+  count = Number(count);
```
Comment (thatblindgeye) [inline 10_fibonacci/solution/fibonacci-solution.js:2]: Nit: we could assign this to a new variable instead of mutating the `count` arg

**Review by thatblindgeye — COMMENTED** [review]
Looks like there's a merge conflict that needs resolving

**Comment by thatblindgeye** [comment]
Closing due to inactivity.

**Pull request closed without merging**